src/lib/operators/join_hash.cpp

```diff
@@ -268,7 +277,7 @@ class JoinHash::JoinHashImpl : public AbstractJoinOperatorImpl {
                                          _mode == JoinMode::AntiNullAsTrue || _mode == JoinMode::AntiNullAsFalse;
 
     // Containers used to store histograms for (potentially subsequent) radix
-    // partitioning phase (in cases _radix_bits > 0). Created during materialization phase.
+    // partitioning stage (in cases _radix_bits > 0). Created during materialization stage.
```

Comment:
I see that either "phase" or "stage" should be used consistently, but just out of interest: Is there any particular reason why "stage" should be preferred over "phase"?

Reply:
Nope. That's simply how the join grew over time. Will change them all to steps. I am fine with Markus's idea of using steps and the hash join was the initial operator why we first thought about such "phases/stages/steps". It's also called `join_hash_steps.hpp`.

Anyways, thanks for pointing it out!